### ISSUE TITLE
Auto Close Function

### ISSUE THREAD

**Issue by goozleology** [rohanb10/snoozz-tab-snoozing#89]
I'd like to be able to set Snoozz to automatically close browser windows at certain times each day.

For example, I need to check my email on a regular basis, so I have set Snoozz to open a browser window with my three email accounts three times per day. The problem is that if I don't close the window, then I end up with three windows with three email accounts open.

Chrome

**Comment by rohanb10** [comment]
Yeah, I get that this is a bit frustrating, but I'm not really looking to introduce any auto-closing functionality for Snoozz. 

1. Closing will be prevented if any of your open tabs have any unsaved data and send out a javascript `confirm(...)` to ask if you're sure that you want to close this webpage
2. Closing tabs based on the URL is will be extremely finicky. Using my gmail as an example, on the inbox page I am on `https://mail.google.com/mail/u/0/#inbox`, so I can set Snoozz to look for tabs with that URL to autoclose. But If I have a email thread, open then the url will become `https://mail.google.com/mail/u/0/#inbox/FMfcgzGmvLVJdTHXzlKPZnQQTpdSTklg` which will not match the original URL.

Basically I think that adding auto-close will add a lot more complications than it will solve, and if there are bugs (which will inevitably come up) the last thing I want to do is deal with user complaints because of data loss.

Again, I understand this is a frustrating situation. My recommendation is for you to just wake up your window of 3 email tabs once a day. Keep a bookmark folder of these three tabs saved in your bookmark bar and just `cmd` or `ctrl` click it when you need them open. You'll be able to open all three tabs in one click. And if you need to open a specific one, then you will be able to open that folder and do it in two clicks
